src/ethereum/index.ts

```diff
@@ -161,8 +162,8 @@ export default class EthereumWallet implements IWallet {
 
   async generateMetaData(
     sdkVersion: string,
-    contractAddress?: string,
-    contractAbbr: string = 'ETH'
+    contractAbbr: string,
```

Comment:
don't flip params, first is optional (nullable and second is default) you can archive required logic without flipping data,

Reply:
the `contractAbbr` is being made mandatory from default. hence the flip is necessary. Will have to check if it is possible to achieve the things without making `contractAbbr` mandatory.